testutil/testmigration/fixtures_recoverable_account.go

```diff
@@ -0,0 +1,646 @@
+		if err := mf.addAccount(i, MorseEOA); err != nil {
+			return err
+		}
+	}
+
+	// Generate module accounts which represent system accounts
+	for i := range mf.config.ValidAccountsConfig.NumModuleAccounts {
+		if err := mf.addAccount(i, MorseModule); err != nil {
+			return err
+		}
+	}
+
+	// Generate invalid accounts with addresses that are too short
+	for i := range mf.config.InvalidAccountsConfig.NumAddressTooShort {
+		if err := mf.addAccount(i, MorseInvalidTooShort); err != nil {
+			return err
+		}
+	}
+
+	// Generate invalid accounts with addresses that are too long
+	for i := range mf.config.InvalidAccountsConfig.NumAddressTooLong {
+		if err := mf.addAccount(i, MorseInvalidTooLong); err != nil {
+			return err
+		}
+	}
+
+	// Generate accounts with invalid hex characters in the address
+	for i := range mf.config.InvalidAccountsConfig.NumNonHexAddress {
+		if err := mf.addAccount(i, MorseNonHex); err != nil {
+			return err
+		}
+	}
+
+	// Application accounts section - Create staked application accounts
+
+	// Generate standard applications with both staked and unstaked accounts
+	for i := range mf.config.ValidAccountsConfig.NumApplications {
+		if err := mf.addApplication(i, MorseApplication); err != nil {
+			return err
+		}
+	}
+
+	// Generate orphaned application accounts without corresponding unstaked accounts
+	for i := range mf.config.OrphanedActorsConfig.NumApplications {
+		if err := mf.addApplication(i, MorseOrphanedApplication); err != nil {
+			return err
+		}
+	}
+
+	// Validator accounts section - Create staked validator accounts
+
+	// Generate standard validators with both staked and unstaked accounts
+	for i := range mf.config.ValidAccountsConfig.NumValidators {
+		if err := mf.addValidator(i, MorseValidator); err != nil {
+			return err
+		}
+	}
+
+	// Generate orphaned validator accounts without corresponding unstaked accounts
+	for i := range mf.config.OrphanedActorsConfig.NumValidators {
+		if err := mf.addValidator(i, MorseOrphanedValidator); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+// addAccount creates and adds an unstaked account to the Morse state export and account state.
+// This function handles different types of accounts, including regular EOAs, module accounts,
+// and accounts with invalid addresses for testing error handling.
+func (mf *MorseMigrationFixtures) addAccount(
+	actorTypeIndex uint64,
+	accountType MorseUnstakedActorType,
+) (err error) {
+	// Get the next global index for this account
+	allActorsIndex := mf.NextIndex()
+
+	// Generate a deterministic private key for this account index
+	privKey := mf.generateMorsePrivateKey(allActorsIndex)
+	pubKey := privKey.PubKey()
+	address := pubKey.Address()
+
+	// Determine the actor type and modify the address if needed based on account type
+	var actorType, moduleAccountName string
+	switch accountType {
+	case MorseEOA:
+		actorType = migrationtypes.MorseExternallyOwnedAccountType
```

Comment:
Ditto for the other cases.

```suggestion
		accountType = migrationtypes.MorseExternallyOwnedAccountType
```